compiler/rustc_middle/src/mir/interpret/error.rs

```diff
@@ -191,9 +192,8 @@ pub enum InvalidProgramInfo<'tcx> {
     FnAbiAdjustForForeignAbi(call::AdjustForForeignAbiError),
     /// SizeOf of unsized type was requested.
     SizeOfUnsizedType(Ty<'tcx>),
-    /// An unsized local was accessed without having been initialized.
-    /// This is not meaningful as we can't even have backing memory for such locals.
-    UninitUnsizedLocal,
+    /// We are runnning into a nonsense situation due to ConstProp violating our invariants.
+    ConstPropNonsense,
```

Comment:
🤣 🔥

Reply:
When furious debugging ended up with an unsized field in a sized type, I might have been a little frustrated ;)